zeebe/zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/SnapshotOnlyDb.java

```diff
@@ -103,4 +112,55 @@ private UnsupportedOperationException unsupported(final String operation) {
         "Failed to execute 'ZeebeDb#%s'; this operation is not supported on a snapshot-only DB"
             .formatted(operation));
   }
+
+  public void copySnapshot(
+      final ZeebeTransactionDb<ZbColumnFamilies> toDB, final Set<ColumnFamilyScope> scopes) {
+    try (final var readOptions = PrefixReadOptions.readOptions()) {
+      final var toCtx = toDB.createContext();
+      final var abort = new AtomicBoolean(false);
```

Comment:
:question: Why is it `AtomicBoolean`? Is there concurrent access to this object? Can use agrona's `MutableBoolean` instead.

Reply:
Good point, I think I added because of the lambda, but forgot to fix it later on :+1: